lib/org_date_rangeable.rb

```diff
@@ -0,0 +1,27 @@
+module OrgDateRangeable
+  def monthly_range(org:, start_date: Date.today.end_of_month, end_date: Date.today.end_of_month)
+    raise ArgumentError.new('missing org') unless org.present?
+    raise ArgumentError.new('missing start_date') unless start_date.respond_to?(:end_of_month)
+    raise ArgumentError.new('missing end_date') unless end_date.respond_to?(:end_of_month)
+
+    where(["org_id = ? and date >= ? and date <= ?", org.id, start_date, end_date]) 
```

Comment:
Prefer named variables in SQL strings:

``` ruby
    where("org_id = :org_id AND date >= :start_date AND :end_date <= ?", org_id: org.id, start_date: start_date, end_date: end_date) 
```

Reply:
it is also good for methods declaration involving more than two arguments...